Sources/SwiftProtobuf/Google_Protobuf_Duration_Extensions.swift

```diff
@@ -157,6 +157,11 @@ public extension Google_Protobuf_Duration {
 
 extension Google_Protobuf_Duration: ExpressibleByFloatLiteral {
     public typealias FloatLiteralType = Double
+    public init(seconds: Int64, nanos: Int32 = 0) {
```

Comment:
I don't see a problem with asking people to write out `Google_Protobuf_Duration(seconds: 0, nano: 12)` in that case.

We could change it later.